packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -26,6 +29,12 @@ function BlockListAppender( {
 	isLocked,
 	renderAppender: CustomAppender,
 } ) {
+	const ref = useRef();
+	const { isDraggingOverElement } = useBlockDropZone( {
+		element: ref,
+		rootClientId,
+	} );
```

Comment:
It's not entirely clear why a dropzone was added to this component, there was none before?

Reply:
I moved it from the child components into here.